### DISCUSSION TITLE
After ROI update, must reselect the device

### DISCUSSION THREAD

**Discussion by wonder99** [springbok/MLM2PRO-GSPro-Connector#38] [General]
I had a problem with my ROI boxes, and after I corrected them, the problem persisted until I selected the same device again.  I suggest the updated ROIs take effect immediately if they were for the selected device.

**Comment by springbok** [comment]
Ok I think I do a reload on device selection, after you updated the ROI's did you select 'Cancel' instead of 'Select'? I'll have a look and reload after 'Save'.
